routing_common/vehicle_model.hpp

```diff
@@ -37,8 +37,11 @@ class IVehicleModel
 
   virtual bool IsOneWay(FeatureType const & f) const = 0;
 
-  /// @returns true iff feature |f| can be used for routing with corresponding vehicle model.
+  /// @returns true if feature |f| can be used for routing with corresponding vehicle model.
```

Comment:
There is no spelling error here.
iff = if and only if
We use the shortened version fairly often.